docs/core/tutorials/snippets/library-with-visual-studio/vb/ShowCase/Program.vb

```diff
@@ -22,7 +22,7 @@ Module Program
             Console.ReadKey()
         End If   
         Console.Clear()
-        Console.WriteLine("{0}Press <Enter> only to exit; otherwise, enter a string and press <Enter>:{0}",
+        Console.WriteLine($"{Environment.NewLine}Press <Enter> only to exit; otherwise, enter a string and press <Enter>:{Environment.NewLine}",
                           vbCrLf)
```

Comment:
line 26 no longer needed

```suggestion
        Console.WriteLine($"{Environment.NewLine}Press <Enter> only to exit; otherwise, enter a string and press <Enter>:{Environment.NewLine}")
```